addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -260,6 +272,13 @@ options.Class.include({
      * @override
      */
     _computeWidgetState: async function (methodName, params) {
+        if (methodName === 'setShadow') {
+            let [shadowColor, shadowX, shadowY, shadowBlur, shadowSpread, shadowInset] = this._getShadowValues(this.$target);
+            return parseInt(shadowX) != 0 ||
+                parseInt(shadowY) != 0 ||
+                parseInt(shadowBlur) != 0 ||
+                parseInt(shadowSpread) != 0
+        }
```

Comment:
The `setShadow` method, if required, should not be available in all options. Create a new JS option to handle it.